sw/device/silicon_creator/lib/sigverify/BUILD

```diff
@@ -78,6 +78,8 @@ opentitan_functest(
     cw310 = cw310_params(
         timeout = "long",
     ),
+    # Temporarily disable this test because it takes too long.
+    tags = ["broken"],
```

Comment:
Actually, instead of marking these `broken` can you set their timeout values so that this change will work with #14003?

Reply:
In fact, #14003 implements a timeout filter, and all these are marked "timeout=long", so they will already be blocked by #14003 and there won't be a need for this PR.

Reply:
Also, it turns out that the `broken` tag doesn't reach the sh_test targets when it is a top-level attribute (instead of in device-specific params). So the wycheproof tests still ended up running :joy: 

Probably we ought to have the device-specific sh_tests inherit tags from the top-level kwargs, but that's for another PR.